tests/unit/python/fledge/services/core/api/test_support.py

```diff
@@ -362,3 +373,338 @@ def mock_syslog(_level):
+            with patch.object(SupportBuilder, "__init__", return_value=None):
+                with patch.object(SupportBuilder, "build", side_effect=mock_build):
+                    # Test manual support bundle creation (which simulates automated creation)
+                    resp = await client.post('/fledge/support')
+                    res = await resp.text()
+                    jdict = json.loads(res)
+                    assert 200 == resp.status
+                    assert "bundle created" in jdict
+                    
+                    # Verify the bundle name format for manual creation (no service name)
+                    bundle_name = jdict["bundle created"]
+                    assert bundle_name == "support-180301-13-35-23.tar.gz"
+
+    async def test_automated_support_bundle_disabled(self, client):
+        """Test that support bundle creation works even when auto_support_bundle is disabled"""
+        from fledge.common.service_record import ServiceRecord
+        
+        # Mock configuration with auto support bundle disabled
+        mock_config = {
+            "auto_support_bundle": {
+                "value": "false",
+                "description": "Automatically create support bundle when service fails",
+                "type": "boolean",
+                "default": "true",
+                "displayName": "Auto Generate On Failure"
+            },
+            "support_bundle_retain_count": {
+                "value": "3",
+                "description": "Number of support bundles to retain (minimum 1)",
+                "type": "integer",
+                "default": "3",
+                "minimum": "1",
+                "displayName": "Bundles To Retain"
+            }
+        }
+        
+        # Mock the build method to return a bundle name
+        async def mock_build(service_name=None):
+            if service_name:
+                return f'support-{service_name}-180301-13-35-23.tar.gz'
+            else:
+                return 'support-180301-13-35-23.tar.gz'
+        
+        # Test that manual support bundle creation still works when auto is disabled
+        with patch.object(support, 'get_support_bundle_config', return_value=mock_config):
+            with patch.object(SupportBuilder, "__init__", return_value=None):
+                with patch.object(SupportBuilder, "build", side_effect=mock_build):
+                    # Test manual support bundle creation
+                    resp = await client.post('/fledge/support')
+                    res = await resp.text()
+                    jdict = json.loads(res)
+                    assert 200 == resp.status
+                    assert "bundle created" in jdict
+
+                    # Verify the bundle name format for manual creation
+                    bundle_name = jdict["bundle created"]
+                    assert bundle_name == "support-180301-13-35-23.tar.gz"
+
+    async def test_support_builder_with_service_name(self, client):
```

Comment:
Is this test case already included in test_automated_support_bundle_on_service_failure? Here, we are solely verifying the support bundle by its name.

We should see parameterized test, review the current tests, and utilize them.